public_html/wp-content/mu-plugins/camptix-tweaks/camptix-tweaks.php

```diff
@@ -497,19 +497,18 @@ function disable_attendee_edits( $attendee ) {
 /**
  * Log when published attendees are cancelled.
  *
- * @param string  $to
- * @param string  $from
  * @param WP_Post $post
  */
-function log_publish_to_cancel( $to, $from, $post ) {
+function log_publish_to_cancel( $post ) {
 	/** @var $camptix CampTix_Plugin */
 	global $camptix;
 
-	if ( 'tix_attendee' !== $post->post_type || $to === $from ) {
+	if ( 'tix_attendee' !== $post->post_type ) {
 		return;
 	}
 
-	if ( 'publish' === $from && 'cancel' === $to ) {
+	// Not a manual change, should be logged.
+	if ( 0 === wp_get_current_user_id() ) {
```

Comment:
```suggestion
	if ( 0 === get_current_user_id() ) {
```

Currently throwing a fatal error here.